indicators/migrations/0013_set_all_calculation_type_to_numeric.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2018-07-04 09:56
+from __future__ import unicode_literals
+
+from django.db import migrations
+from ..models import Indicator
+
+
+def set_calculation_type(apps, schema_editor):
+    for indicator in Indicator.objects.all():
+        indicator.calculation_type = Indicator.CALC_TYPE_NUMERIC
+        indicator.save()
```

Comment:
If you are just updating one field that for every object will be the same, I'd recommend you to do better `Indicator.objects.all().update(calculation_type=Indicator.CALC_TYPE_NUMERIC)`. This is way more efficient.

Reply:
Done!